src/coreclr/tools/aot/ILCompiler.LLVM/ILCompiler.LLVM.csproj

```diff
@@ -67,6 +63,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="libLLVM" Version="11.0.0" />
+    <PackageReference Include="LLVMSharp" Version="15.0.0-beta1" />
```

Comment:
What's the difference between libLLVM and LLVMSharp?

Reply:
libLLVM contains `libLLVM.dll` (via the `runtime.json` and the libllvm.runtime.win-x64 package) which is the native LLVM library for writing LLVM bitcode.  LLVMSharp is the object oriented interface over libLLVM (via the interop package) that Ilc uses in the IL backend.  When we delete the IL backend, we can remove LLVMSharp.